docs/process.rst

```diff
@@ -134,20 +134,29 @@ To use a language other than Bash, simply start your process script with the cor
 `shebang <http://en.wikipedia.org/wiki/Shebang_(Unix)>`_. For example::
 
     process perlStuff {
+        input:
+        val(x)
+        
         """
         #!/usr/bin/perl
 
-        print 'Hi there!' . '\n';
+        print 'value: ${x}';
         """
     }
 
     process pythonStuff {
+        input:
+        path(x)
+        
+        output:
+        path "table.txt"
+    
         """
         #!/usr/bin/python
-
-        x = 'Hello'
-        y = 'world!'
-        print "%s - %s" % (x,y)
+        // Reads the content of a file, and writes it to `table.txt`
```

Comment:
```suggestion
        # Reads the content of a file, and writes it to `table.txt`
```

It's Python now, not Groovy/Nextflow so we should use the appropriate comment symbol.